api/basic.yaml

```diff
@@ -11,10 +11,18 @@ info:
     email: snlab.org@gmail.com
 paths: {}
 definitions:
+  ResourceID:
+    pattern: ^[0-9a-zA-Z-:@_\.]{1,64}$
+    type: string
+  CostMetric:
+    pattern: ^[0-9a-zA-Z-:@_\.]{1,64}$
+    type: string
+  CostMode:
+    type: string
```

Comment:
Just a minor comment: The base ALTO protocol definitely accepts either `numerical` or `ordinal` as the valid **cost-mode**. Other valid **cost-mode** values are defined in the extensions. So maybe we can use the `oneOf` keyword defined in JSONSchema V4 to enable the inheritance.